src/clj/clojurians_log/views.clj

```diff
@@ -74,13 +103,18 @@
      [:div.message {:id (cl.tu/format-inst-id inst)}
       [:a.message_profile-pic {:href "" :style (str "background-image: url(" image-48 ");")}]
       [:a.message_username {:href ""} name]
-      [:span.message_timestamp [:a {:href (str "#" (cl.tu/format-inst-id inst))} (cl.tu/format-inst-time inst)]]
+      [:span.message_timestamp [:a {:href (->> (url hostname
+                                                    (:channel/name channel)
+                                                    date
+                                                    (cl.tu/format-inst-id (:message/inst message)))
```

Comment:
Does this need to be a fully qualified url (http://foo.bar/channel/...) or would a relative url do as well (/channel/message/ts)?

By the way https://github.com/lambdaisland/uri can deal well with relative urls ;)

Reply:
Well, the FB open graph object debugger complains about it if it's not a fully qualified url. I'm not sure if it is simply a way to simplify storage/caching the meta data or if there are other concerns.